typescript/ws-protocol-examples/README.md

```diff
@@ -1,22 +1,36 @@
-# Foxglove Studio WebSocket protocol examples
+# Foxglove WebSocket examples in TypeScript
 
-This package provides example server and client scripts using the [Foxglove Studio WebSocket protocol](https://github.com/foxglove/ws-protocol).
+The Foxglove WebSocket protocol is encoding-agnostic, as long as the desired encoding is supported by both client and server. It currently supports Protobuf and JSON messages.
 
-## Usage
+## Installation
+
+Install the `ws-protocol-examples` `npm` package:
+
+```
+$ npm install @foxglove/ws-protocol-examples
+```
+
+Run example servers `sysmon` and `image-server` using the following commands:
 
 ```
-$ npx @foxglove/ws-protocol-examples --help
-$ npx @foxglove/ws-protocol-examples image-server
 $ npx @foxglove/ws-protocol-examples sysmon
-$ npx @foxglove/ws-protocol-examples simple-client [host]
+$ npx @foxglove/ws-protocol-examples image-server
 ```
 
-To see data from the example servers, open up https://studio.foxglove.dev and initiate a Foxglove WebSocket connection to `ws://localhost:8765/`.
+To see the data transmitted by one of these servers, open [studio.foxglove.dev](https://studio.foxglove.dev) in a browser, and initiate a Foxglove WebSocket connection to the respective WebSocket URL.
+
+<img width="500" alt="Foxglove Studio displaying memory and CPU usage from the system monitor example" src="https://user-images.githubusercontent.com/14237/145313065-85c05645-6b29-4eb2-a498-849c83f8792d.png">
+
+You can also run a simple example client that subscribes to messages on all channels with the `json` encoding.
+
+```
+$ npx @foxglove/ws-protocol-examples simple-client localhost:8765
+```
 
 ## Development
 
-Note: This package lives inside a monorepo which uses [yarn workspaces](https://yarnpkg.com/features/workspaces), so most commands (other than `yarn install`) should be prefixed with `yarn workspace @foxglove/ws-protocol-examples ...`.
+The `ws-protocol-examples` package lives inside a monorepo that uses [yarn workspaces](https://yarnpkg.com/features/workspaces), so most commands (other than `yarn install`) should be prefixed with `yarn workspace @foxglove/ws-protocol-examples...`.
```

Comment:
Space was intentional, though we could remove the `...` if you find it strange. Just wanted to show that it is meant to be a prefix on other commands.